Source/Core/DolphinQt/CMakeLists.txt

```diff
@@ -151,13 +274,109 @@ PRIVATE
   imgui
 )
 
+set_target_properties(dolphin-emu PROPERTIES PROJECT_LABEL "Dolphin")
+
 if(WIN32)
-  target_sources(dolphin-emu PRIVATE DolphinQt.manifest)
+  if (CMAKE_BUILD_TYPE STREQUAL "Debug")
+    set(Dolphin_NAME "DolphinD")
+  else()
+   set(Dolphin_NAME "Dolphin")
+  endif()
+
+  set_target_properties(dolphin-emu PROPERTIES
+    WIN32_EXECUTABLE TRUE
+    OUTPUT_NAME ${Dolphin_NAME}
+  )
+  target_sources(dolphin-emu PRIVATE DolphinQt.manifest DolphinQt.rc)
+  target_compile_options(dolphin-emu PRIVATE /D _SILENCE_CXX17_RESULT_OF_DEPRECATION_WARNING)
+
+  # Copy files for Windows
+
+  # Copy Sys directory
+  set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS "${CMAKE_SOURCE_DIR}/Data/Sys")
+  file(GLOB_RECURSE resources RELATIVE "${CMAKE_SOURCE_DIR}/Data" "${CMAKE_SOURCE_DIR}/Data/Sys/*")
+  foreach(res ${resources})
+    configure_file("${CMAKE_SOURCE_DIR}/Data/${res}" "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Release/${res}" COPYONLY)
```

Comment:
If I'm reading this correctly, it's creating duplicates of the Sys folder in Debug and Release? The current behavior is to have both the debug and release binaries in the same directory, with different names.

Reply:
This causes a variety of issues, namely given the fact that we expect that debug DLLs won't be present in the binary directory unless you explicitly do a debug build. This is not easily doable with cmake.